wpiunits/src/main/java/edu/wpi/first/units/Units.java

```diff
@@ -250,6 +250,23 @@ private Units() {
   public static final Mult<Mult<Mass, Distance>, Distance> KilogramSquareMeters =
       Kilograms.mult(Meters).mult(Meters);
 
+  // Force
+  /** The base SI unit for force. */
+  public static final Mult<Mass, Velocity<Velocity<Distance>>> Newtons =
+      Kilograms.mult(Meters.per(Second).per(Second));
+
+  /** Singular alias for Newtons. */
+  public static final Mult<Mass, Velocity<Velocity<Distance>>> Newton = Newtons; // alias;
+
+  // Torque
+  /** The base SI unit for torque. */
+  public static final Mult<Mult<Mass, Velocity<Velocity<Distance>>>, Distance> NewtonMeters =
+      Newtons.mult(Meters);
```

Comment:
See above re: aliasing

Reply:
Gotcha.  At first I thought you meant the variable name.